Jenkinsfile

```diff
@@ -32,6 +32,7 @@ pipeline {
     stage("Build") {
       steps {
         sh "npm --unsafe-perm install"
```

Comment:
❔I thought `cypress install` was already happening postinstall?

Reply:
Fair, let's remove it. Thanks for pointing this!